doc/content/source/kernels/MFEMVectorFEDomainLFKernel.md

```diff
@@ -0,0 +1,27 @@
+# MFEMVectorFEDomainLFKernel
+
+!syntax description /Kernels/MFEMVectorFEDomainLFKernel
+
+## Overview
+
+Adds the domain integrator for integrating the linear form
+
+!equation
+(\vec f, \vec v)_\Omega \,\,\, \forall \vec v \in V
```

Comment:
Add description of v

Reply:
Added, along with a description of $\vec f$